src/services/cards.ts

```diff
@@ -413,7 +413,7 @@ export class CardsService {
     if (globalTags) {
       for (let i = 0; i < globalTags.length; i++) {
         globalTags[i] = globalTags[i].replace("#", "");
-        globalTags[i] = globalTags[i].replace("/", "::");
+        globalTags[i] = globalTags[i].replaceAll("/", "::");
```

Comment:
This is not a solution because the method replaceAll does not exist. The proper one is:
`globalTags[i] = globalTags[i].replace(/\//g, "::");`